### DISCUSSION TITLE
animation debug supports multi character

### DISCUSSION THREAD

**Discussion by ghost** [Vortex2Oblivion/LeatherEngine-Extended-Support#4] [Ideas]
would make positioning the characters better
also the image is fake lol, its just what i think it would look like
![funi](https://github.com/Vortex2Oblivion/LeatherEngine-Extended-Support/assets/77215994/64c3d84b-af7f-4c52-9358-c266bb0de9e4)

**Comment by Vortex2Oblivion** [comment]
not a bad idea, ill look into it
